### PR TITLE
Custom Logging Path

### PR DESCRIPTION
**Pull request by ennerperez**

### CODE REVIEW

File: LibreHardwareMonitor/Utilities/Logger.cs [hunk 1]
```diff
@@ -80,9 +81,18 @@ private void SensorRemoved(ISensor sensor)
...
         }
 
-        private static string GetFileName(DateTime date)
+        private string GetFileName(DateTime date)
         {
-            return AppDomain.CurrentDomain.BaseDirectory + Path.DirectorySeparatorChar + string.Format(FileNameFormat, date);
+            string pathName = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, LoggingPath);
+            try
+            {
+                if (Directory.Exists(pathName)) Directory.CreateDirectory(pathName);
```
Comment (PhyxionNL) [inline LibreHardwareMonitor/Utilities/Logger.cs:89]: Please put this on two lines to match code stile.

File: LibreHardwareMonitor/UI/MainForm.cs [hunk 2]
```diff
@@ -279,6 +279,7 @@ public MainForm()
             authWebServerMenuItem.Checked = _settings.GetValue("authenticationEnabled", false);
 
             _logSensors = new UserOption("logSensorsMenuItem", false, logSensorsMenuItem, _settings);
+            _logger.LoggingPath = _settings.GetValue("logginPath", "Logs");
```
Comment (PhyxionNL) [inline LibreHardwareMonitor/UI/MainForm.cs:282]: "loggin" should be "logging"

**Review by PhyxionNL — CHANGES_REQUESTED** [review]

File: LibreHardwareMonitor/Utilities/Logger.cs (hunk 1 above)
Comment (ennerperez) [reply LibreHardwareMonitor/Utilities/Logger.cs:89]: OK

File: LibreHardwareMonitor/UI/MainForm.cs (hunk 2 above)
Comment (ennerperez) [reply LibreHardwareMonitor/UI/MainForm.cs:282]: Sorry. Ok

**Review by ennerperez — COMMENTED** [review]
Reviewed changes

**Comment by PhyxionNL** [comment]
Wouldn't it make sense to make this work with an absolute/relative path instead? I don't know how useful it would be to make it always relative to the app folder.

**Comment by ennerperez** [comment]
There are some scenarios where it can be useful, in my case I want to port all to a custom folder for using PowerBi to have a metric over time, but leaving it in the root of the app makes it difficult to read.

On the other hand, perhaps by customizing this path, the logs can be saved in another location, like a network path, or a different path from the app, to have all the logs in a better and safe place.

On the other hand, in the case of building an installer in %programfiles% the path where the app is not writable, so the logs must go to %appdata%, or another folder defined by the user, this must be considered for make the change in the UI, but for now being able to configure the route is quite useful.

**Comment by PhyxionNL** [comment]
I was more thinking in terms of something that if the logpath is specified, that's used as-is, otherwise, do the existing behavior.

**Comment by ennerperez** [comment]
Anyways, It's not recommended to put log in same app folder, kind of disorder.

**Comment by JIGASHI** [comment]
Where is even the default log location?

**Pull request closed without merging**